Makefile

```diff
@@ -192,8 +192,37 @@ $(BINDIR)/e2e.test: .init
 	$(DOCKER_CMD) $(BINDIR)/openapi-gen \
 		--v 1 --logtostderr \
 		--go-header-file "vendor/github.com/kubernetes/repo-infra/verify/boilerplate/boilerplate.go.txt" \
-		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1,k8s.io/client-go/pkg/api/v1,k8s.io/apimachinery/pkg/apis/meta/v1" \
+		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1" \
+		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/settings/v1alpha1" \
+		--input-dirs "k8s.io/apimachinery/pkg/apis/meta/v1" \
+		--input-dirs "k8s.io/api/core/v1" \
 		--output-package "github.com/kubernetes-incubator/service-catalog/pkg/openapi"
+	# Generate defaults
```

Comment:
I think a comment delineating that the remaining generator calls are for the settings API would be helpful